packages/core/src/Stores/notification-store.js

```diff
@@ -362,7 +363,11 @@ export default class NotificationStore extends BaseStore {
             } else {
                 this.removeNotificationByKey({ key: this.client_notifications.poi_dob_mismatch });
             }
-
+            if (is_poa_older_than_six_months) {
+                this.addNotificationMessage(this.client_notifications.poa_older_than_six_months);
+            } else {
+                this.removeNotificationByKey({ key: this.client_notifications.poa_older_than_six_months });
+            }
             if (loginid !== LocalStore.get('active_loginid')) return;
```

Comment:
we need to add the notification to `is_non_hidden_notification` in app-notifications to display the notification in mobile